tests/commands/try_repo_test.py

```diff
@@ -98,6 +101,37 @@ def test_try_repo_relative_path(cap_out, tempdir_factory):
         assert not try_repo(try_repo_opts(relative_repo, hook='bash_hook'))
 
 
+def test_try_repo_no_commits_no_git(cap_out, tempdir_factory):
+    repo = tempdir_factory.get()
+    with pytest.raises(CalledProcessError) as e:
+        try_repo(try_repo_opts(repo, hook='bash_hook'))
+    assert 'appears to have no commits' not in str(e.value)
```

Comment:
I don't really see how this is a useful test -- if you're going to catch an exception you should assert what it actually outputs rather than what it _doesn't_ output -- there's an infinite number of things it doesn't output but ideally only one that it actually does